Build Issue  - Version 0.18.2

# Unable to Build
## build command gives error - js/app.059374b8.js from Terser
Unexpected character '@' [external "@fortawesome/vue-fontawesome":1,0][js/app.059374b8.js:212,17]

## Issue Command
 "npm run build"
## Expected behaviour
Previous versions build fine
## Software used
Mac OS - Mojave
## Other information
 Building for production...

 ERROR  Failed to compile with 1 errors                                                                                                                   11:07:02 AM

 error  

js/app.059374b8.js from Terser
Unexpected character '@' [external "@fortawesome/vue-fontawesome":1,0][js/app.059374b8.js:212,17]
    at j (/Users/pksingh/procmodeler/node_modules/terser/dist/bundle.min.js:1:18809)
    at m (/Users/pksingh/procmodeler/node_modules/terser/dist/bundle.min.js:1:20120)
    at Object.ee [as input] (/Users/pksingh/procmodeler/node_modules/terser/dist/bundle.min.js:1:25867)
    at a (/Users/pksingh/procmodeler/node_modules/terser/dist/bundle.min.js:1:27285)
    at s (/Users/pksingh/procmodeler/node_modules/terser/dist/bundle.min.js:1:27339)
    at mt (/Users/pksingh/procmodeler/node_modules/terser/dist/bundle.min.js:1:49467)
    at Mt (/Users/pksingh/procmodeler/node_modules/terser/dist/bundle.min.js:1:49608)
    at T (/Users/pksingh/procmodeler/node_modules/terser/dist/bundle.min.js:1:33415)
    at /Users/pksingh/procmodeler/node_modules/terser/dist/bundle.min.js:1:29785
    at /Users/pksingh/procmodeler/node_modules/terser/dist/bundle.min.js:1:28257

 ERROR  Build failed with errors.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @processmaker/modeler@0.18.2 build: `vue-cli-service build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @processmaker/modeler@0.18.2 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/pksingh/.npm/_logs/2020-01-26T16_07_02_527Z-debug.log

--- a/README.md
+++ b/README.md
@@ -34,10 +34,7 @@ You can now run any of the below commands. Most commands make calls `vue-cli-ser
 # Compile the app and start a local development server
 npm run serve
 
-# Create a production build of the app
-npm run build
-
-# Compile the app to be distributed as an npm package
+# Create a production build of the app to be distributed as an npm package
 npm run build-bundle
 
 # Report and fix ESLint errors

--- a/package.json
+++ b/package.json
@@ -3,7 +3,6 @@
   "version": "0.18.2",
   "scripts": {
     "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
     "open-cypress": "TZ=UTC cypress open",
     "lint": "vue-cli-service lint --no-fix",
     "build-bundle": "vue-cli-service build --target lib --name modeler ./src/components/nodes/index.js",

